gppt/_selenium.py

```diff
@@ -176,22 +174,35 @@ def __try_login(self) -> None:
             )
 
     @staticmethod
-    def __get_headless_option() -> webdriver.chrome.options.Options:
+    def __get_chrome_option(headless: bool | None) -> OptionsType:
         options = webdriver.ChromeOptions()
-        options.add_argument("--headless")
-        options.add_argument("--disable-gpu")
-        options.add_argument("--disable-extensions")
-        options.add_argument("--disable-infobars")
-        options.add_argument("--disable-dev-shm-usage")
-        options.add_argument("--disable-browser-side-navigation")
-        options.add_argument('--proxy-server="direct://"')
-        options.add_argument("--proxy-bypass-list=*")
-        options.add_argument("--start-maximized")
-        options.add_argument("--no-sandbox")
-        options.add_argument("--disable-dev-shm-usage")
-        options.add_argument("--user-agent=" + USER_AGENT)
-        options.add_experimental_option("excludeSwitches", ["enable-automation"])
-        options.add_experimental_option("useAutomationExtension", False)
+
+        if headless:
+            options.add_argument("--headless")
+            options.add_argument("--disable-gpu")
+            options.add_argument("--disable-extensions")
+            options.add_argument("--disable-infobars")
+            options.add_argument("--disable-dev-shm-usage")
+            options.add_argument("--disable-browser-side-navigation")
+            options.add_argument('--proxy-server="direct://"')
+            options.add_argument("--proxy-bypass-list=*")
```

Comment:
These lines should be removed. Now when you try to login pixiv with headless mode, the `--proxy-server` and `--proxy-bypass` arguments will be added twice.

```python
options.add_argument('--proxy-server="direct://"')
options.add_argument("--proxy-bypass-list=*")
```

Reply:
After remove these lines, now `gppt lh` is okay to me 😄

Reply:
Thanks. Fixed: 279783258327d1bc36107d9022df11d14003edb4